### DISCUSSION TITLE
will explorer patcher be updated to work with  windows 11 again. it stopped working correctly after latest windows 11 update.

### DISCUSSION THREAD

**Discussion by agar37** [valinet/ExplorerPatcher#3013] [Q&A]
my taskbar.. which was actually 2 levels high.  is now transparent. and it interferes with me reading whats on the taskbar .. as whats behind it on the desktop mixes with the taskbar. but taskbar functionality does not seem to work now. the only reason i am using windows 11 is because of explorer patcher and now it doesnt work .

**Comment by pyrates999** [comment]
Fixed in the current release of EP already.  Please update to it.

If you have classic shell installed, uninstall it. That is no longer maintained.

The replacement is open shell.

**Reply by MeMyselfAndAI2023** [reply]
Thank you soooo much for the quick fix to the issue after Win11 update! Awesome! I can confirm it works as expected again! 🥳👍👍👍

**Reply by pyrates999** [reply]
Please set my response as the answer then.

**Reply by MeMyselfAndAI2023** [reply]
Sorry, I don't know how to do that here...?

**Reply by pyrates999** [reply]
Click here and see if it has an option to set this as the answer:

![image](https://github.com/valinet/ExplorerPatcher/assets/106135298/887bb753-8288-4d2e-9913-c82c2933d8c2)

**Reply by MeMyselfAndAI2023** [reply]
Unfortunately not, no such option there.
![image](https://github.com/valinet/ExplorerPatcher/assets/126958707/7bb81a77-18f4-4632-b969-72b82ae1371f)

**Reply by pyrates999** [reply]
Here's how to do it: https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions

You should see a mark as answer below my comment above.

**Reply by MeMyselfAndAI2023** [reply]
No.
Your help doc link says „ You can mark a comment in the discussion as an answer to the discussion **if a discussion is within a category that accepts answers.**“
Maybe that‘s the reason why.
I checked 4 times now to mark as answer to help you, but will not try it a fifth time cause there is simply no such option. Sorry.

**Reply by pyrates999** [reply]
Ah ok.  Thanks for trying :)

**Comment by agar37** [comment]
i got it updated.. thanks much :) you guys are awesome
    On Sunday, March 17, 2024 at 07:19:29 AM EDT, pyrates999 ***@***.***> wrote:  
 
 


Here's how to do it: https://docs.github.com/en/discussions/managing-discussions-for-your-community/moderating-discussions

You should see a mark as answer below my comment above.
